<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VirtualDriveFrame</class>
 <widget class="QFrame" name="VirtualDriveFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>67</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Shape::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Shadow::Sunken</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,0">
   <item row="0" column="1">
    <widget class="QLabel" name="dirPath">
     <property name="text">
      <string notr="true">error</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <widget class="QPushButton" name="dirDelete">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="dirLetter"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="dirWinLabel"/>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QPushButton" name="dirChange">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
